### PR TITLE
add sc24 page, without floormap

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Looks good to me. I defer

File: content/pages/sc24.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            may mean for integrations to come.
+            </p>
+
+            <p>
+            In addition, join us for troubleshooting time with the experts.  Bring us your tricky questions and
+            weird scenarios.  Come learn from others.
+            </p>
+
+            <p>
+            The iRODS Consortium was started as an open-source software development organization in 2013 by
```
Comment (korydraughn) [inline content/pages/sc24.html:45]: `open-source` on this line has a hyphen while the one on L27 does not.

File: content/pages/sc24.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            <br>9:30 - 11:30 a.m. ET
+            <br>Peachtree Offices at Downtown Atlanta
+            <br>235 Peachtree St, Suite 400
+            <br>Atlanta, Georgia 30303
+
+                <div>
+                <a href="https://forms.gle/tJEF7uhQNYbTGEr67">
+                    <div class="btn btn-success">RSVP to save your seat at the free iRODS SC24 Mini-Workshop</div>
+                </a>
+                </div>
```
Comment (korydraughn) [inline content/pages/sc24.html:63]: Indentation needs attention for L59-L63.

File: content/pages/sc24.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<html>
+<head><title>sc24</title></head>
```
Comment (korydraughn) [inline content/pages/sc24.html:2]: `sc24` -> `SC24` ?

**Review by korydraughn — COMMENTED** [review]
The wording is fine.

I noticed some spots where periods are followed by two spaces and other locations where periods are followed by one space.

Other than that, seems good.

File: content/pages/sc24.html (hunk 3 above)
Comment (trel) [reply content/pages/sc24.html:2]: all others have been lowercase... i don't think this actually renders anywhere...   look at https://irods.org/sc23 

resolving.

File: content/pages/sc24.html (hunk 1 above)
Comment (trel) [reply content/pages/sc24.html:45]: ah, yes, removed the hyphen.

File: content/pages/sc24.html (hunk 2 above)
Comment (trel) [reply content/pages/sc24.html]: fixed

**Comment by trel** [comment]
whitespace renders equally - but made them all double (i like typewriters).

should be ready again.

File: content/pages/sc24.html (hunk 3 above)
Comment (korydraughn) [reply content/pages/sc24.html:2]: `<title>` elements are rendered in the tab name.

Looking at the HTML for SC23's page, it puts "iRODS" in the tab name as defined by the `<title>` element.

Unless the tool used to publish the page modifies the HTML, I'm betting `sc24` may appear in the tab name.

**Review by korydraughn — APPROVED** [review]
Looks ready.

Squash and pound if you're happy with it.

**Comment by trel** [comment]
squashed - ready to merge.

**Comment by korydraughn** [comment]
Merge it.

**Pull request merged**